.travis.yml

```diff
@@ -19,4 +19,4 @@ before_install:
   - tar -xzf ../apache-cassandra-2.1.12-bin.tar.gz -C ../
   - sh ../apache-cassandra-2.1.12/bin/cassandra > /dev/null
 
-script: sh test/utils/run_tests.sh coverage all && (npm run-script coveralls || exit 0)
+script: (sh test/utils/run_tests.sh coverage all && npm run-script coveralls || exit 0)
```

Comment:
The way I read it, this will always return 0 no matter what. Is that what we want? :)